src/components/Button/index.tsx

```diff
@@ -36,8 +37,9 @@ const Button: FC<ButtonProps> = ({
       rounded={rounded}
     >
       <ButtonWrapper
-        hasBorder={hasBorder}
+        hasBorder={hasBorder || variant === 'outline'}
```

Comment:
acho que agora nao precisamos mais da prop "hasBorder", podemos validar pelo tipo do botao, exibir a bordar se for outline